.github/workflows/build.yml

```diff
@@ -1,14 +1,16 @@
 name: build
 on: [workflow_dispatch, push]
 env:
+  PRD_NAME: "consul-lambda"
   PKG_NAME: "consul-lambda-registrator"
```

Comment:
Maybe this would be clearer as `REGISTRATOR_NAME` or `REG_NAME`?

Reply:
Agreed. I've changed it to `REG_NAME` as suggested.